hs_core/views/utils.py

```diff
@@ -663,9 +663,6 @@ def link_irods_file_to_django(resource, filepath, overwrite=False):
         ret = None
         try:
             ret = ResourceFile.get(resource=resource, file=base, folder=folder)
-            if overwrite:
-                ret.delete()
```

Comment:
This was deleting the overwritten file.
https://github.com/hydroshare/hydroshare/blob/develop/hs_core/models.py#L2616

Reply:
overwrite was removed as a parameter in the method.  Basically, instead of deleting the ResourceFile and then creating another one, we're just reusing the existing ResourceFile.